### PR TITLE
Adding support for adoption scenario 1

### PR DESCRIPTION
**Pull request by averdagu**
Dataplane adoption docs supports doing the adoption while wallaby environment uses different subnets than next generation. This commit enables Makefiles in order to prepare the environment with different subnets between environments.

Related: [OSPRH-5881](https://issues.redhat.com//browse/OSPRH-5881)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,22 @@ CLEANUP_DIR_CMD					 ?= rm -Rf
 # be also cloned from OPENSTACK_REPO/OPENSTACK_BRANCH.
 CHECKOUT_FROM_OPENSTACK_REF	?= false
 
+ADOPTED_SCENARIO					?=false
```
Comment (booxter) [inline Makefile:38]: names of variables here should not refer to adoption. There's a general case for deploying CP and DP services in different ranges. We should be able to stand up greenfield deployment with different ranges.

File: scripts/gen-nncp.sh [hunk 2]
```diff
@@ -145,6 +133,25 @@ EOF_CAT
...
+        next-hop-interface: ${INTERFACE}.${internalapi_vlan_id}
+      - destination: ${STORAGE_PREFIX}.0/24
+        next-hop-interface: ${INTERFACE}.${storage_vlan_id}
+      - destination: ${STORAGEMGMT_PREFIX}.0/24
+        next-hop-interface: ${INTERFACE}.${storagemgmt_vlan_id}
+      - destination: ${TENANT_PREFIX}.0/24
+        next-hop-interface: ${INTERFACE}.${tenant_vlan_id}
+EOF_CAT
+    fi
+    if [ -n "$IPV6_ENABLED" ]; then
```
Comment (booxter) [inline scripts/gen-nncp.sh:149]: hm. why is it needed?

**Review by booxter — CHANGES_REQUESTED** [review]
Could you please give a playbook of commands that one would use to deploy the setup as you intend? Please add them to commit message.

Please also fix pre-commits.

**Comment by averdagu** [comment]
recheck

**Comment by averdagu** [comment]
recheck

**Comment by averdagu** [comment]
recheck

**Comment by danpawlik** [comment]
recheck

**Review by booxter — APPROVED** [review]

**Pull request merged**